### PR TITLE
c# on mac

### PR DESCRIPTION
**Pull request by anvuong87**

### CODE REVIEW

File: csharp_mac/Utils/Capabilities.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        {
+            capabilities_.SetCapability("username", env_.GetEnvironmentInfo().USER_NAME);
+            capabilities_.SetCapability("accessKey", env_.GetEnvironmentInfo().API_KEY);
+            capabilities_.SetCapability("sessionName", "[C#] Android Web");
+            capabilities_.SetCapability("sessionDescription", "testing selenium web with c# on android device");
+            capabilities_.SetCapability("deviceOrientation", "portrait");
+            capabilities_.SetCapability("captureScreenshots", true);
+            capabilities_.SetCapability("deviceGroup", "KOBITON");
+            capabilities_.SetCapability("browserName", "chrome");
+            capabilities_.SetCapability("deviceGroup", "KOBITON");
```
Comment (khanhdodang) [inline csharp_mac/Utils/Capabilities.cs]: Remove deviceGroup

File: csharp_mac/Utils/Capabilities.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            capabilities_.SetCapability("platformName", "Android");
+            capabilities_.SetCapability("deviceName", "*");
+            capabilities_.SetCapability("platformVersion", "*");
+
+
+            // capabilities_.SetCapability("automationName", "UIAutomator2");
```
Comment (khanhdodang) [inline csharp_mac/Utils/Capabilities.cs]: Remove

File: python/README.md [hunk 3]
```diff
@@ -57,13 +57,16 @@ $ rm -rf *.testlog
```
Comment (khanhdodang) [inline python/README.md]: Remove https://github.com/kobiton/samples/commit/6f4dcdfe41891309c0a8f5bff917b5e02e746907#diff-7141304209be3da2a29ef8b0b79886fd

File: csharp_mac/Utils/Capabilities.cs (hunk 1 above)
Comment (anvuong87) [reply csharp_mac/Utils/Capabilities.cs]: fixed

File: csharp_mac/Utils/Capabilities.cs (hunk 2 above)
Comment (anvuong87) [reply csharp_mac/Utils/Capabilities.cs]: fixed

**Pull request merged**